### PR TITLE
feat(topbar/user): added the possibility to render custom content. Ad…

### PR DESCRIPTION
**Pull request by devictoribero**
This component is only used for `Coches` and `Motos` right now.

- Added the possibility to render custom content on the right instead of the `navCTA`.
- Added tests

Screen Shot 2020-07-28 at 11 25 57

### CODE REVIEW

**Comment by devictoribero** [comment]
which one do you mean @andresin87 ? any suggestion tho?

File: components/topbar/user/src/index.js [hunk 1]
```diff
@@ -225,19 +223,22 @@ export default function TopbarUser({
...
+      {customContent}
+      {navCTA && !customContent && (
+        <div className="sui-TopbarUser-ctaButton">
+          <AtomButton
+            link
+            href={navCTA.url}
+            title={navCTA.text}
+            leftIcon={<navCTA.icon svgClass="sui-TopbarUser-ctaButtonIcon" />}
+            size="small"
+            type="primary"
```
Comment (jordevo) [inline components/topbar/user/src/index.js:235]: 🍆 I know it's unrelated to this `PR` but perhaps you could use size and type values as imported from `AtomButton`?

File: components/topbar/user/src/index.js [hunk 2]
```diff
@@ -347,6 +348,11 @@ TopbarUser.propTypes = {
       })
     )
   }).isRequired,
+  /**
+   * Render custom content instead of a CTA.
+   * If a customContent is provided, it's rendered instead of a navCTA
+   */
+  customContent: PropTypes.node,
```
Comment (jordevo) [inline components/topbar/user/src/index.js:355]: 🍆 just wondering... would you see any benefit in using `children` instead of this `customContent` prop?

Comment (devictoribero) [reply components/topbar/user/src/index.js:355]: IMHO children should be used when you want to place something inside, it's a too strict API. @jordevo

File: components/topbar/user/src/index.js (hunk 1 above)
Comment (devictoribero) [reply components/topbar/user/src/index.js:235]: I can, yes

File: components/topbar/user/src/index.js (hunk 2 above)
Comment (jordevo) [reply components/topbar/user/src/index.js:355]: fair enough! 👍

**Pull request merged**